docs/dpdk/README.md

```diff
@@ -15,6 +15,8 @@ For more information on hugepage support in Kubernetes please see [here](https:/
 ### VF drivers
 DPDK applications require devices to be attached with supported dpdk backend driver.
 For Intel® x700 series NICs `igb_uio` or `vfio-pci` is required.
+For Mellanox ConnectX®-4 Lx EN Adapter `mlx5_core` or `mlx5_ib` is required.
+For Mellanox ConnectX®-5 Adapter `mlx5_core` or `mlx5_ib` is required.
```

Comment:
perhaps make these bullets and put both connectX4 and 5 in the same bullet e.g:

### VF drivers
DPDK applications require devices to be attached with supported dpdk backend driver.
- For Intel® x700 series NICs `igb_uio` or `vfio-pci` is required.	For Intel® x700 series NICs `igb_uio` or `vfio-pci` is required.
- For Mellanox ConnectX®-4 Lx,  ConnectX®-5 Adapters `mlx5_core` or `mlx5_ib` is required.